external/storm-elasticsearch/src/main/java/org/apache/storm/elasticsearch/bolt/EsLookupBolt.java

```diff
@@ -76,12 +78,12 @@ public void process(Tuple tuple) {
     }
 
     private Collection<Values> lookupValuesInEs(Tuple tuple) throws IOException {
-    	String index = tupleMapper.getIndex(tuple);
-    	String type = tupleMapper.getType(tuple);
-    	String id = tupleMapper.getId(tuple);
-    	Map<String, String> params = tupleMapper.getParams(tuple, new HashMap<>());
+        String index = tupleMapper.getIndex(tuple);
+        String type = tupleMapper.getType(tuple);
+        String id = tupleMapper.getId(tuple);
+        Map<String, String> params = tupleMapper.getParams(tuple, new HashMap<>());
```

Comment:
should indent with Tab

Reply:
hi Danny, we might be not allowed to indent with tab because of FileTabCharacter in checkstyle. Please correct me if I misunderstood here.

Reply:
It was broken indentation (mixed tab and space) and the change fixes the indentation.